src/Frame.cc

```diff
@@ -72,9 +66,7 @@ Frame::Frame(uint32_t session_id, carta::FileLoader* loader, const std::string&
     std::string log_message;
     if (!_loader->FindCoordinateAxes(_image_shape, _spectral_axis, _stokes_axis, log_message)) {
         _open_image_error = fmt::format("Problem determining file shape: {}", log_message);
-        if (_verbose) {
-            carta::Log(session_id, _open_image_error);
-        }
+        spdlog::error("Session {}: {}", session_id, _open_image_error);
```

Comment:
This one should be a warning.

Reply:
actually I think it should be an error, because it's returning straight after that, so the image will not be loaded successfully.

Reply:
Oh, good point. In that case this is fine.